.github/actions/build-vars/action.yml

```diff
@@ -4,12 +4,19 @@ description: 'Variables shared by build workflows'
 outputs:
   env_name:
     value: ${{ steps.build_result.outputs.env_name }}
+    description: 'Test, staging, or prod'
   version:
     value: ${{ steps.version_result.outputs.version }}
+    description: 'Tag name'
   has_router_change:
     value: ${{ steps.backend_change_result.outputs.has_router_change }}
+    description: 'For non-prod, do not build if no changes'
   has_react_change:
     value: ${{ steps.frontend_change_result.outputs.has_react_change }}
+    description: 'For non-prod, do not build if no changes'
```

Comment:
maybe "do not build frontend if no changes" ?

Reply:
Prod builds are set to always create a build so that build artifacts are generated in the release. If I don't build, I could probably fetch the previous artifact, but it may have also been deleted/expired.